docs/apps/transfer/authorizations.md

```diff
@@ -0,0 +1,49 @@
+# TransferAuthorization
+
+`TransferAuthorization` implements the `Authorization` interface for `ibc.applications.transfer.v1.Msg`. It allows a granter to grant a grantee the privilege to submit MsgTransfer on its behalf. Please see the [Cosmos SDK docs](https://docs.cosmos.network/v0.47/modules/authz) for more details on granting privileges via the `Authz` module.
+
+More specifically, the granter allows the grantee to transfer funds that belong to the granter on a set of 1 or more source port ID/channel ID pairs.
+
+For each source port ID/channel ID pair, the granter shall be able to specify a spend limit for each denomination they wish to allow the grantee to be able to transfer.
+
+The granter shall be able to specify the list of addresses that they allow to receive funds. If empty, then all addresses are allowed.
+
+
+It takes: 
+
+- a range of `SourcePorts` and a range of `SourceChannels` which together comprise the unique transfer channel identifiers over which authorized funds can be transferred.
```

Comment:
the term `range` sounds like you are referring to the older version of the code where there were multiple slices passed (all different slices), this has been refactored to use [this type](https://github.com/cosmos/ibc-go/blob/294522546a10ecf993fb35257dc949d5b3bccc60/modules/apps/transfer/types/authz.pb.go#L29-L39)

And a `TransferAuthorization` is created like [this](https://github.com/cosmos/ibc-go/blob/b4bd0a79435a5da164d1fd1a6c56a54887d6683b/modules/apps/transfer/types/transfer_authorization.go#L19-L23)